apps/pdc-dashboard/config/sync/core-store.plugin_content_manager_configuration_content_types##api##product.product.json

```
@@ -220,6 +220,20 @@
           "sortable": true
         }
       },
+      "oldSlugs": {
+        "edit": {
+          "label": "Vorige Slugs",
+          "description": "Bewaart de eerder URL's van de inhoud voor tracking of omleidingen.",
```

```suggestion
          "description": "Bewaart de eerdere URL's van de inhoud voor tracking of omleidingen.",
```

Deze zin is misschien een beetje verwarrend. De naamgeving in het label is `Slugs` maar in de description is het `URLs`. 

"Lijst van oude URL's om te herleiden naar deze pagina"?